auth/src/main/java/com/firebase/ui/auth/provider/GoogleProvider.java

```diff
@@ -40,15 +41,19 @@
     private Activity mActivity;
     private IDPCallback mIDPCallback;
 
-    public GoogleProvider(Activity activity, IDPProviderParcel parcel) {
+    public GoogleProvider(Activity activity, IDPProviderParcel parcel, @Nullable String email) {
         mActivity = activity;
         String mClientId = parcel.getProviderExtra().getString(CLIENT_ID_KEY);
         GoogleSignInOptions googleSignInOptions;
 
-        googleSignInOptions = new GoogleSignInOptions.Builder(GoogleSignInOptions.DEFAULT_SIGN_IN)
+        GoogleSignInOptions.Builder builder = new GoogleSignInOptions.Builder(GoogleSignInOptions
+                .DEFAULT_SIGN_IN)
                 .requestEmail()
-                .requestIdToken(mClientId)
-                .build();
+                .requestIdToken(mClientId);
+        if (email != null) {
```

Comment:
We had another issue where email was coming through as "".  Maybe use TextUtils.isEmpty here.